from exception import AuthException


class InvalidPasswordException(AuthException):
    pass